activerecord/lib/active_record.rb

```diff
@@ -336,12 +336,12 @@ def self.global_executor_concurrency # :nodoc:
   singleton_class.attr_accessor :dump_schemas
   self.dump_schemas = :schema_search_path
 
-  ##
-  # :singleton-method:
-  # Show a warning when Rails couldn't parse your database.yml
-  # for multiple databases.
-  singleton_class.attr_accessor :suppress_multiple_database_warning
-  self.suppress_multiple_database_warning = false
+  def self.suppress_multiple_database_warning=(value)
+    ActiveSupport::Deprecation.warn(<<-MSG.squish)
+      config.suppress_multiple_database_warning= is deprecated and will be removed in Rails 7.2.
+      It no longer has any effect and should be removed from the configuration file.
+    MSG
+  end
```

Comment:
I think we need the getter deprecated as well as the setter

Reply:
Good call! Even if the getter is not used anymore... I just force-pushed again with the deprecated getter method.